site/FlexAIDdS/theme.css

```
@@ -0,0 +1,150 @@
+    background-image 300ms ease,
+    border-color 300ms ease,
+    color 300ms ease,
+    fill 300ms ease,
+    box-shadow 300ms ease;
+}
+
+/* ── Canonical pill toggle ──────────────────────────────────── */
+.lbp-theme-toggle {
+  /* Accent resolves to whatever the host page calls its accent. */
+  --lbp-accent: var(--color-accent, var(--teal, var(--neon-b, #22D3EE)));
+  appearance: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)